src/test/kotlin/test/assertk/assertions/InputStreamSpec.kt

```diff
@@ -0,0 +1,90 @@
+package test.assertk.assertions
+
+import assertk.assert
+import assertk.assertions.hasNotSameContentAs
+import assertk.assertions.hasSameContentAs
+import org.assertj.core.api.Assertions.*
+import org.jetbrains.spek.api.Spek
+import org.jetbrains.spek.api.dsl.given
+import org.jetbrains.spek.api.dsl.it
+import java.io.ByteArrayInputStream
+import java.io.InputStream
+import java.util.*
+
+class InputStreamSpec: Spek ({
+
+    given("an empty stream") {
+
+        it("has the same content as another empty stream") {
+            assert(emptyStream()).hasSameContentAs(emptyStream())
+        }
+
+        it("has the same content as another empty stream (using hasNotSameContentAs)") {
+            assertThatThrownBy {
+                assert( emptyStream() ).hasNotSameContentAs( emptyStream() )
```

Comment:
Can you fix your formatting? The current style doesn't have spaces around fun args. If you are using intellij the format code option should fix things.